broker-core/src/main/java/io/zeebe/broker/workflow/processor/subprocess/TriggerStartEventHandler.java

```diff
@@ -70,7 +70,7 @@ public void handle(BpmnStepContext<ExecutableFlowElementContainer> context) {
     context.getOutput().appendNewEvent(WorkflowInstanceIntent.EVENT_TRIGGERING, value);
   }
 
-  private boolean isNoneStartEvent(ExecutableCatchEventElement startEvent) {
+  public static boolean isNoneStartEvent(ExecutableCatchEventElement startEvent) {
```

Comment:
I think it's better to move this logic into `ExecutableCatchEventElement` instead of having a dependency between the handlers.